### DISCUSSION TITLE
practice and remember

### DISCUSSION THREAD

**Discussion by ghost** [Cyfrin/foundry-full-course-cu#2004] [General]
can you please tell me how can i or any student practice this course and remember all of these and how can i male notes for practicing and make some projects 
if you tell me this will help me or others

**Comment by EngrPips** [comment]
Hello @rudcr7, There is no shortcut to mastery. You need to practice! practice!! And practice!! About not taking I just create a `note.md` file in each project and put all my notes there.

**Reply by krishno16** [reply]
how can i write the notes i mean to say from video or anywhere else

**Reply by EngrPips** [reply]
I watched the video and made my notes from the video. Also, there is also a written lesson for every video so maybe you can draw inspiration from that.
